test/highlight/highlight_test.dart

```diff
@@ -0,0 +1,7 @@
+void main() {
+// <- type
```

Comment:
These comments are actually test assertions: https://tree-sitter.github.io/tree-sitter/syntax-highlighting#unit-testing

`<-` means go to the last non-comment line, and check the symbol at the column where this comment starts
`^` means go to the last non-comment line, and check the symbol at the column where this `^` is